organization/utils/__init__.py

```diff
@@ -0,0 +1,56 @@
+from django.core.cache import cache
+from django.db.models import Case, Count, ExpressionWrapper, F, Max, When
+from django.db.models.fields import FloatField
+from django.utils import timezone
+from django.utils.translation import gettext as _, gettext_noop
+
+from userprofile.models import UserProfile
+from problem.models import Problem
+from submission.models import Submission
+
+CACHE_DURATION = 86400
+
+def organizations_of_member(profile: UserProfile):
+  key = 'org_member:%d' % profile.id
```

Comment:
`org_member` means Organizations where the user is a member, or members of the Organization? The name might be a little bit confusing..